packages/components/checkbox/src/checkbox-group.vue

```diff
@@ -1,85 +1,70 @@
-<script lang="ts">
-import {
-  computed,
-  defineComponent,
-  h,
-  nextTick,
-  provide,
-  renderSlot,
-  toRefs,
-  watch,
-} from 'vue'
+<template>
+  <component
+    :is="tag"
+    :id="groupId"
+    :class="ns.b('group')"
+    role="group"
+    :aria-label="!isLabeledByFormItem ? label || 'checkbox-group' : undefined"
+    :aria-labelledby="isLabeledByFormItem ? elFormItem.labelId : undefined"
+  >
+    <slot />
+  </component>
+</template>
+
+<script lang="ts" setup>
+import { computed, nextTick, provide, toRefs, watch } from 'vue'
 import { UPDATE_MODEL_EVENT } from '@element-plus/constants'
 import { debugWarn } from '@element-plus/utils'
 import { useNamespace, useSize } from '@element-plus/hooks'
 import {
+  checkboxEmits,
   useCheckboxGroup,
   useCheckboxGroupId,
   useCheckboxGroupProps,
 } from './checkbox'
 
-export default defineComponent({
+defineOptions({
   name: 'ElCheckboxGroup',
+})
 
-  props: useCheckboxGroupProps,
-
-  emits: [UPDATE_MODEL_EVENT, 'change'],
-
-  setup(props, { emit, slots }) {
-    const { elFormItem } = useCheckboxGroup()
-    const { groupId, isLabeledByFormItem } = useCheckboxGroupId(props, {
-      elFormItem,
-    })
-    const checkboxGroupSize = useSize()
-    const ns = useNamespace('checkbox')
-
-    const changeEvent = (value) => {
-      emit(UPDATE_MODEL_EVENT, value)
-      nextTick(() => {
-        emit('change', value)
-      })
-    }
+const props = defineProps(useCheckboxGroupProps)
+const emit = defineEmits(checkboxEmits)
 
-    const modelValue = computed({
-      get() {
-        return props.modelValue
-      },
-      set(val) {
-        changeEvent(val)
-      },
-    })
+const { elFormItem } = useCheckboxGroup()
+const { groupId, isLabeledByFormItem } = useCheckboxGroupId(props, {
+  elFormItem,
+})
+const checkboxGroupSize = useSize()
+const ns = useNamespace('checkbox')
 
-    provide('CheckboxGroup', {
-      name: 'ElCheckboxGroup',
-      modelValue,
```

Comment:
Why remove this?

Reply:
Since `modelValue` comes from `props.modelValue`, the original eslint error prompts `Incorrect 'modelValue' is specified more than once, so this usage will be overwritten`.

Reply:
He passed the verification of the action, and I have restored it.